tests/test_pipreqs.py

```diff
@@ -37,50 +47,51 @@ def setUp(self):
+        cls.non_existent_filepath = os.path.join(cls.project, "non_existent_file.txt")
+
+        cls.parsed_packages = [
             {"name": "pandas", "version": "2.0.0"},
             {"name": "numpy", "version": "1.2.3"},
             {"name": "torch", "version": "4.0.0"},
         ]
 
-        self.parsed_packages_no_version = [
+        cls.parsed_packages_no_version = [
             {"name": "pandas", "version": None},
             {"name": "tensorflow", "version": None},
             {"name": "torch", "version": None},
         ]
 
-        self.parsed_packages_any_version = [
+        cls.parsed_packages_any_version = [
             {"name": "numpy", "version": None},
             {"name": "pandas", "version": "2.0.0"},
             {"name": "tensorflow", "version": None},
             {"name": "torch", "version": "4.0.0"},
         ]
 
-        self.project_clean = os.path.join(os.path.dirname(__file__), "_data_clean")
-        self.project_invalid = os.path.join(os.path.dirname(__file__), "_invalid_data")
-        self.parsed_packages = [
-            {"name": "pandas", "version": "2.0.0"},
-            {"name": "numpy", "version": "1.2.3"},
-            {"name": "torch", "version": "4.0.0"},
-        ]
-        self.empty_filepath = os.path.join(self.project, "empty.txt")
-        self.imports_filepath = os.path.join(self.project, "imports.txt")
-
-        self.project_with_ignore_directory = os.path.join(os.path.dirname(__file__), "_data_ignore")
-        self.project_with_duplicated_deps = os.path.join(os.path.dirname(__file__), "_data_duplicated_deps")
-
-        self.requirements_path = os.path.join(self.project, "requirements.txt")
-        self.alt_requirement_path = os.path.join(self.project, "requirements2.txt")
-        self.non_existing_filepath = "xpto"
+        cls.project_clean = os.path.join(os.path.dirname(__file__), "_data_clean")
+        cls.project_invalid = os.path.join(os.path.dirname(__file__), "_invalid_data")
+        cls.project_with_ignore_directory = os.path.join(os.path.dirname(__file__), "_data_ignore")
+        cls.project_with_duplicated_deps = os.path.join(os.path.dirname(__file__), "_data_duplicated_deps")
+
+        cls.requirements_path = os.path.join(cls.project, "requirements.txt")
+        cls.alt_requirement_path = os.path.join(cls.project, "requirements2.txt")
+        cls.non_existing_filepath = "xpto"
+
+        cls.project_with_notebooks = os.path.join(os.path.dirname(__file__), "_data_notebook")
+        cls.project_with_invalid_notebooks = os.path.join(os.path.dirname(__file__), "_invalid_data_notebook")
+        cls.compatible_files = {
+            "original": os.path.join(os.path.dirname(__file__), "_data/test.py"),
+            "notebook": os.path.join(os.path.dirname(__file__), "_data_notebook/test.ipynb"),
+        }
+        cls.requirements_notebook_path = os.path.join(cls.project_with_notebooks, "requirements.txt")
```

Comment:
is this variable being defined here but already used in the last commit? this need to be fixed

Reply:
Fixed.